server/GQLFactory/helpers/typeHelpers.js

```diff
@@ -73,10 +73,47 @@ mutationsHelper.mutationFields = (
 
 /*   Functions facilitating creation of custom types */
 const customHelper = {};
+/*  Loops through SQL columns getting their name as fieldName, type, and isNullable to be returned as fields */
+customHelper.getFields = (primaryKey, foreignKeys, columns) => {
+  let fields = ``;
 
-customHelper.getFields = () => {};
+  for (const fieldName of Object.keys(columns)) {
+    if (!(foreignKeys && foreignKeys[fieldName]) && fieldName !== primaryKey) {
+      const { dataType, columnDefault, isNullable } = columns[fieldName];
+      fields += `\n  ${fieldName}: ${typeConversion[dataType]}`;
+      if (isNullable === 'NO' && columnDefault === null) fields += `!`;
+    }
+  }
+  return fields;
+};
+
+customHelper.getRelationships = (tableName, sqlSchema) => {
+  let relationshipFields = ``;
+  const tableData = sqlSchema[tableName];
+  const { foreignKeys, referencedBy } = tableData;
 
-customHelper.getRelationships = () => {};
+  if (foreignKeys) {
```

Comment:
Can we write a comment to explain which relationships this conditional is accounting for?